src/content/en/2021/third-parties.md

```diff
@@ -1,20 +1,491 @@
+
+One difference that we have taken this year is to remove the Hosting category from our data this year. If you happen to use Wordpress.com for your blog, or Shopify for your ecommerce platform, then we're going to ignore other requests for those domains by that site as not truly "third-party", as they are in many ways part of hosting on those platforms. Similar to the way we noted above, we do not consider CDNs in front of a page to be "third party".
+
+In reality this made very little difference to the numbers, but we feel it's a more accurate reflection of what we should consider "third party" by above definition, and also aligns more closely with how the other chapters use this term.
+
+### Caveats
+
+- All data presented here is based on a non-interactive, cold load. These values could start to look quite different after user interaction.
+- The pages are tested from servers in the US with no cookies set, so third-parties requested after opt-in are not included. This will especially affect pages hosted and predominantly served to countries in scope for the <a href="https://en.wikipedia.org/wiki/General_Data_Protection_Regulation">General Data Protection Regulation</a>, or other similar legislation.
+- Only the home pages are tested. Other pages may have different third-party requirements.
+- Some of the lesser-used third-party domains are uncategorised into the _unknown_ category. As part of this analysis we submitted more categories for the top used domains to improve this data set.
+
+Learn more about our [methodology](./methodology).
+
+## Prevalence
+
+So how much are third parties used? Well, the answer is a lot!
+
+{{ figure_markup(
+  caption="Percentage of mobile sites using at least one third-party resource.",
+  content="94.40%",
+  classes="big-number",
+  sheets_gid="1082574922",
+  sql_file="percent_of_websites_with_third_party.sql"
+)
+}}
+
+A staggering 94.40% of mobile sites, and 94.07% of desktop sites use at least one third-party resource. Even with our newer restrictive definition of third-parties this represents a continued growth from when the Web Almanac started in 2019.
+
+{{ figure_markup(
+  image="third-parties-websites-using-third-parties-by-year.png",
+  caption="Websites using third-parties by year.",
+  description="Bar chart showing the number of websites using third parties over the last three years, split by desktop and mobile. On desktop 89.1% of websites used a third party in 2019, followed by 93.9% in 2020, and 94.1% in 2021. On mobile 88.5% used a third party in 2019, followed by 94.0% in 2020, and 94.4% in 2021",
+chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkT-CF5-NB7Oatd6XZq_08EoMGfiygKZtW4OwVivaaW3cIlt3ZcWNBtOdNePD_nKzkvb2nMlhWOX6g/pubchart?oid=1728778967&format=interactive",
+  sheets_gid="1082574922",
```

Comment:
But how can I do that within one graph and keep the label consistent?

And not entirely convinced about your graph. Looks a bit grey scale and difficult to differentiate between then.

Reply:
It would probably require one chart for desktop and one for mobile.

What I was going for in the chart was that these are all mobile stats but they fade out the further away from this year you get.

Feel free to ignore.

Reply:
The fading does introduce colour contrast issues (a touchy subject as we've seen!)

I'm gonna ignore if you don't mind.